### PR TITLE
fix: add dataSelection option

### PR DESCRIPTION
**Pull request by jobo322**
with it is it possible to priorize one over the other FT/FID

remove keepFT n keepFID

### CODE REVIEW

**Comment by lpatiny** [comment]
After discussion with @targos it would make more sense to remove the properties 'keepFID' and 'keepFT' in favour of:

- dataSelection: 'fid' | 'ft' | 'both' | 'preferFT' | 'preferFID'

File: package.json [hunk 1]
```diff
@@ -49,5 +49,8 @@
     "is-any-array": "^2.0.0",
     "jcampconverter": "^9.0.7",
     "ml-spectra-processing": "^11.16.0"
+  },
+  "volta": {
+    "node": "18.14.0"
```
Comment (lpatiny) [inline package.json:54]: Why are you forcing volta version ? This should not be there I think.

File: src/groupByExperiments.js [hunk 2]
```diff
@@ -150,8 +151,12 @@ export function groupByExperiments(fileCollection, options = {}) {
...
     }
 
-    if (ftFidSelection === 'fid' && hasRawData) continue;
-
+    if (
+      dataSelection === 'fid' ||
+      (dataSelection === 'preferFID' && hasRawData)
+    )
+      continue;
+    console.log('pasa')
```
Comment (lpatiny) [inline src/groupByExperiments.js:159]: Console.log should disappar and the '.only' on the testcase as well.

**Review by lpatiny — CHANGES_REQUESTED** [review]
If this is a draft please take the habit to `gh pr create --fill --draft`

File: package.json (hunk 1 above)
Comment (jobo322) [reply package.json]: my mistake, sorry

File: brukerconverter.d.ts [hunk 3]
```diff
@@ -56,7 +46,7 @@ export interface GroupByExperimentsOptions {
    * priories processed or raw data if both exists
```
Comment (lpatiny) [inline brukerconverter.d.ts:46]: Comment does not seem correct

**Pull request merged**